Modules/posixmodule.c

```diff
@@ -15062,7 +15078,7 @@ os_getrandom_impl(PyObject *module, Py_ssize_t size, int flags)
 }
 #endif   /* HAVE_GETRANDOM_SYSCALL */
 
-#ifdef MS_WINDOWS
+#if defined(MS_WINDOWS) && !defined(MS_WINDOWS_NON_DESKTOP)
```

Comment:
This should be `!defined(MS_WINDOWS_GAMES)`.

The SDK supports `AddDllDirectory()` and `RemoveDllDirectory()` for `WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP | WINAPI_PARTITION_SYSTEM)`.